src/main/java/seedu/address/model/person/Person.java

```diff
@@ -121,6 +119,22 @@ public Set<Tag> getTags() {
         return Collections.unmodifiableSet(tags);
     }
 
+    /**
+     * Changes the lead potential of the client.
+     */
+    public void setLead(Lead newLead) {
+        this.lead = newLead;
+    }
+
+    /**
+     * Adds a set of interactions to the person.
+     *
+     * @param interactions the set of interaction to be added
+     */
+    public void addInteractions(Set<Interaction> interactions) {
```

Comment:
The change to return void instead of interactions is approved. However, notice of such modifications (in the PR) would be appreciated.